.travis/test/smoke

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -e
+
+docker-compose --file docker-compose.yaml up -d web
+docker-compose --file docker-compose.yaml exec app bin/console --version
+ping=$(curl -sS http://localhost:8080/ping 2>&1)
+[[ "$ping" == "pong" ]]
+docker-compose --file docker-compose.yaml down
```

Comment:
https://github.com/libero/walking-skeleton/blob/cf092f08f38d77c3fd046e1fd25399150f0a6c5d/examples.sh#L35-L41 would be better as this won't be run on a failure?

Reply:
Can do, although it gets complex quickly and doesn't allow local debugging to reproduce a failure (as everything is cleaned up on failure)